### ISSUE TITLE
No support for special characters?

### ISSUE THREAD

**Issue by sjenset** [nystudio107/craft3-typogrify#5]
I'm using this on a website, and whenever the filter encounters a word/sentence starting with a Norwegian character it seems to output nothing at all.

Ex. `<h3>'Årø Bilsenter'|typogrify</h3>` results in the following HTML: `<h3></h3>`

````
<pre>{{ dump('Årø Bilsenter') }}</pre>
<pre>{{ dump('Årø Bilsenter'|typogrify) }}</pre>
````

Output:

````
string(15) "Årø Bilsenter"

object(Twig_Markup)#997 (2) {
  ["content":"Twig_Markup":private]=>
  string(0) ""
  ["charset":"Twig_Markup":private]=>
  string(5) "UTF-8"
}
````

**Comment by sjenset** [comment]
Seems like this issue is not caused by your plugin, but rather originates from the \mundschenk-at\php-typography package, so I will open [an issue there](https://github.com/mundschenk-at/php-typography/issues/65).

**Comment by mundschenk-at** [comment]
The issue is fixed in version 6.1.3 of `mundschenk-at/php-typography`.

**Comment by sjenset** [comment]
Awesome :)
